.golangci.yml

```diff
@@ -0,0 +1,54 @@
+run:
+  timeout: 10m
+  issues-exit-code: 1
+  tests: true
+  skip-dirs-use-default: true
+  modules-download-mode: readonly
+  allow-parallel-runners: false
+
+linters:
+  fast: false
+  enable:
+  - gofmt
+  - goimports
+  - golint
+  - gosec
+  - govet
+  - misspell
+  disable-all: false
+  presets:
+  - bugs
+  - unused
+
+# all available settings of specific linters
+linters-settings:
+  gofmt:
+    # simplify code: gofmt with `-s` option, true by default
+    simplify: true
+  goimports:
+    local-prefixes: sigs.k8s.io/service-apis
+  golint:
+    min-confidence: 0.9
```

Comment:
If I'm reading correctly, the [default value here is 0.8](https://github.com/golang/lint/blob/master/golint/golint.go#L23), why set a higher value here?

Reply:
the previous script used 0.9